### ISSUE TITLE
Backwalls - MacOs crash

### ISSUE THREAD

**Issue by Berkays** [aki-art/ONI-Mods#33]
Couldn't pinpoint the exact reason maybe something is conflicting?

V1.1.3

```
Exception while loading mod Backwalls at /Users/rubacava/Library/Application Support/unity.Klei.Oxygen Not Included/mods/Steam/2921809973. 
System.Reflection.ReflectionTypeLoadException: Exception of type 'System.Reflection.ReflectionTypeLoadException' was thrown.
  at (wrapper managed-to-native) System.Reflection.Assembly.GetTypes(System.Reflection.Assembly,bool)
  at System.Reflection.Assembly.GetTypes () [0x00000] in <695d1cc93cca45069c528c15c9fdd749>:0 
  at KMod.DLLLoader.LoadDLLs (KMod.Mod ownerMod, System.String harmonyId, System.String path, System.Boolean isDev) [0x0010c] in <8de6cd11d86441fcb8d144b8d4b27c43>:0 

  at UnityEngine.Debug.LogError (System.Object message, UnityEngine.Object context) [0x00000] in <704cac2d02a9488e8e7e54f3a503296d>:0 
  at Debug.LogError (System.Object obj, UnityEngine.Object context) [0x00000] in <7273b301087c4ae79b7148cae73dd755>:0 
  at DebugUtil.LogErrorArgs (UnityEngine.Object context, System.Object[] objs) [0x00000] in <7273b301087c4ae79b7148cae73dd755>:0 
  at DebugUtil.LogException (UnityEngine.Object context, System.String errorMessage, System.Exception e) [0x00000] in <7273b301087c4ae79b7148cae73dd755>:0 
  at KMod.DLLLoader.LoadDLLs (KMod.Mod ownerMod, System.String harmonyId, System.String path, System.Boolean isDev) [0x00000] in <8de6cd11d86441fcb8d144b8d4b27c43>:0 
  at KMod.Mod.Load (KMod.Content content) [0x00000] in <8de6cd11d86441fcb8d144b8d4b27c43>:0 
  at KMod.Manager.Load (KMod.Content content) [0x00000] in <8de6cd11d86441fcb8d144b8d4b27c43>:0 
  at Global.Awake () [0x00000] in <8de6cd11d86441fcb8d144b8d4b27c43>:0 
  at UnityEngine.Object.Internal_InstantiateSingleWithParent_Injected (UnityEngine.Object data, UnityEngine.Transform parent, UnityEngine.Vector3& pos, UnityEngine.Quaternion& rot) [0x00000] in <704cac2d02a9488e8e7e54f3a503296d>:0 
  at UnityEngine.Object.Internal_InstantiateSingleWithParent (UnityEngine.Object data, UnityEngine.Transform parent, UnityEngine.Vector3 pos, UnityEngine.Quaternion rot) [0x00000] in <704cac2d02a9488e8e7e54f3a503296d>:0 
  at UnityEngine.Object.Instantiate (UnityEngine.Object original, UnityEngine.Vector3 position, UnityEngine.Quaternion rotation, UnityEngine.Transform parent) [0x00000] in <704cac2d02a9488e8e7e54f3a503296d>:0 
  at UnityEngine.Object.Instantiate[T] (T original, UnityEngine.Vector3 position, UnityEngine.Quaternion rotation, UnityEngine.Transform parent) [0x00000] in <704cac2d02a9488e8e7e54f3a503296d>:0 
  at Util.KInstantiate (UnityEngine.GameObject original, UnityEngine.Vector3 position, UnityEngine.Quaternion rotation, UnityEngine.GameObject parent, System.String name, System.Boolean initialize_id, System.Int32 gameLayer) [0x00000] in <7273b301087c4ae79b7148cae73dd755>:0 
  at Util.KInstantiate (UnityEngine.GameObject original, UnityEngine.GameObject parent, System.String name) [0x00000] in <7273b301087c4ae79b7148cae73dd755>:0 
  at LaunchInitializer.Update () [0x00000] in <8de6cd11d86441fcb8d144b8d4b27c43>:0 
Build: U44-537329-S
```

Log: 
[player.log](https://github.com/aki-art/ONI-Mods/files/10509053/player.log)

**Comment by pawelad** [comment]
Same thing happened to me (also v1.1.3.0):

```
Exception while loading mod Backwalls at /Users/pawelad/Library/Application Support/unity.Klei.Oxygen Not Included/mods/Steam/2921809973. 
System.Reflection.ReflectionTypeLoadException: Exception of type 'System.Reflection.ReflectionTypeLoadException' was thrown.
  at (wrapper managed-to-native) System.Reflection.Assembly.GetTypes(System.Reflection.Assembly,bool)
  at System.Reflection.Assembly.GetTypes () [0x00000] in <695d1cc93cca45069c528c15c9fdd749>:0 
  at KMod.DLLLoader.LoadDLLs (KMod.Mod ownerMod, System.String harmonyId, System.String path, System.Boolean isDev) [0x0010c] in <b96779738f7847af9f57ccd250a0f729>:0 

  at UnityEngine.Debug.LogError (System.Object message, UnityEngine.Object context) [0x00000] in <704cac2d02a9488e8e7e54f3a503296d>:0 
  at Debug.LogError (System.Object obj, UnityEngine.Object context) [0x00000] in <1207e456885845d9b53379b4b66fa181>:0 
  at DebugUtil.LogErrorArgs (UnityEngine.Object context, System.Object[] objs) [0x00000] in <1207e456885845d9b53379b4b66fa181>:0 
  at DebugUtil.LogException (UnityEngine.Object context, System.String errorMessage, System.Exception e) [0x00000] in <1207e456885845d9b53379b4b66fa181>:0 
  at KMod.DLLLoader.LoadDLLs (KMod.Mod ownerMod, System.String harmonyId, System.String path, System.Boolean isDev) [0x00000] in <b96779738f7847af9f57ccd250a0f729>:0 
  at KMod.Mod.Load (KMod.Content content) [0x00000] in <b96779738f7847af9f57ccd250a0f729>:0 
  at KMod.Manager.Load (KMod.Content content) [0x00000] in <b96779738f7847af9f57ccd250a0f729>:0 
  at Global.Awake () [0x00000] in <b96779738f7847af9f57ccd250a0f729>:0 
  at UnityEngine.Object.Internal_InstantiateSingleWithParent_Injected (UnityEngine.Object data, UnityEngine.Transform parent, UnityEngine.Vector3& pos, UnityEngine.Quaternion& rot) [0x00000] in <704cac2d02a9488e8e7e54f3a503296d>:0 
  at UnityEngine.Object.Internal_InstantiateSingleWithParent (UnityEngine.Object data, UnityEngine.Transform parent, UnityEngine.Vector3 pos, UnityEngine.Quaternion rot) [0x00000] in <704cac2d02a9488e8e7e54f3a503296d>:0 
  at UnityEngine.Object.Instantiate (UnityEngine.Object original, UnityEngine.Vector3 position, UnityEngine.Quaternion rotation, UnityEngine.Transform parent) [0x00000] in <704cac2d02a9488e8e7e54f3a503296d>:0 
  at UnityEngine.Object.Instantiate[T] (T original, UnityEngine.Vector3 position, UnityEngine.Quaternion rotation, UnityEngine.Transform parent) [0x00000] in <704cac2d02a9488e8e7e54f3a503296d>:0 
  at Util.KInstantiate (UnityEngine.GameObject original, UnityEngine.Vector3 position, UnityEngine.Quaternion rotation, UnityEngine.GameObject parent, System.String name, System.Boolean initialize_id, System.Int32 gameLayer) [0x00000] in <1207e456885845d9b53379b4b66fa181>:0 
  at Util.KInstantiate (UnityEngine.GameObject original, UnityEngine.GameObject parent, System.String name) [0x00000] in <1207e456885845d9b53379b4b66fa181>:0 
  at LaunchInitializer.Update () [0x00000] in <b96779738f7847af9f57ccd250a0f729>:0 
Build: U45-546664-S
```

**Comment by pawelad** [comment]
Let me know if I can help in any way : )
